openedx/frontend-app-admin-portal
# feat: adds flex groups dropdown menu in assignment modal

# Description
Adds support for admins to select flex groups from the dropdown menu to the Learner credit assignment modal. The members associated with that group will be added to the summary tab once the user clicks the `Apply selections` button.
If a group does not have any flex groups, the dropdown will not be shown. 

https://2u-internal.atlassian.net/browse/ENT-9491

https://github.com/user-attachments/assets/29560c1b-c6c4-409d-ab9b-36e346b786ae


# Test scenarios
1. checkout this branch and visit customer page https://localhost.stage.edx.org:1991/alc-general/admin/learner-credit/7f344aea-c28c-4ef0-95e6-fb6e58f98495/assign to choose one of the courses to assign learners to. 
2. verify that if the user clicks on a group but does not the `Apply selections` button and navigates away from the dropdown menu, the selections will clear. The same should apply when the user unchecks a group that was previously added to the summary tab -- previous selections/unselections should persist until the user clicks the 'Apply selections' button.


# For all changes


# Only if submitting a visual change
